libs/p11-kit/patches/010-uclibc.patch

```diff
@@ -0,0 +1,15 @@
+--- a/common/compat.c
++++ b/common/compat.c
+@@ -97,8 +97,12 @@
+ #include <unistd.h>
+ 
+ #if defined (HAVE_PROGRAM_INVOCATION_SHORT_NAME) && !HAVE_DECL_PROGRAM_INVOCATION_SHORT_NAME
++#ifdef __UCLIBC__
++extern const char *program_invocation_short_name;
```

Comment:
What is the actual issue solved here? If const is ok here, it makes sense to have it for all cases not just of uclibc.

Reply:
It's a char in glibc and musl. See: http://downloads.openwrt.org/snapshots/faillogs/arc_arc700/packages/p11-kit/compile.txt